src/main/java/bio/terra/cli/command/shared/options/GroupMember.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.cli.command.shared.options;
+
+import bio.terra.cli.service.SamService;
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the options for `terra groups` commands that manage group
+ * members.
+ *
+ * <p>This class is meant to be used as a @CommandLine.Mixin.
+ */
+public class GroupMember extends GroupName {
```

Comment:
This fails the "is-a" test for me.

Reply:
Changed to `GroupMemberProperties`

Reply:
To satisfy the "is-a", should `GroupName` -> `GroupProperties` too (even though it does only contain "name"...)?

Reply:
I think the "is-a" test is for inheritance/interfaces (e.g. `Oak extends Tree` -> oak is a tree) specifically, not for class names in general. Rules aside though, I prefer to have just `GroupName` here because I think that's more descriptive that `GroupProperties`, since it only contains the name as you say.

Reply:
OK I tried a different approach. Instead of using inheritance, I nested the mixin classes (so `GroupMember` contains a `GroupName` property instead of extending it). I also did this for the other set of mixins that used inheritance (`ResourceName`, `CreateResource`, `CreateControlledResource`).